app/src/main/res/layout/item_frequently_asked_question_card.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/root"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <com.kickstarter.ui.views.FrequentlyAskedQuestionCard
+        android:id="@+id/question_answer_layout"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        app:frequently_asked_answer_text="Yes, you can add any additional items through our pledge manager."
```

Comment:
nit: Can we delete the hardcoded texts here? it can be initialized to empty strings if you like, but not need to have them on the xml's can lead to bugs displaying those text like happen before on the AddOns card for the limits

Reply:
sure☺️